shiny/ui/_chat.py

```diff
@@ -1325,6 +1414,209 @@ async def _send_custom_message(self, handler: str, obj: ClientMessage | None):
             },
         )
 
+    def enable_bookmarking(
+        self,
+        client: ClientWithState | chatlas.Chat[Any, Any],
+        /,
+        *,
+        on: Optional[Literal["response"]] = "response",
+    ) -> CancelCallback:
+        """
+        Enable bookmarking for the chat instance.
+
+        This method registers `on_bookmark` and `on_restore` hooks on `session.bookmark`
+        to save/restore chat state on both the `Chat` and `client` instances.
+        In order for this method to actually work correctly, a `bookmark_store`
+        must be specified within `shiny.App()` or `shiny.express.app_opts()`.
+
+        Parameters
+        ----------
+        client
+            The chat client instance to use for bookmarking. This can be a Chat model
+            provider from [chatlas](https://posit-dev.github.io/chatlas/), or more
+            generally, an instance following the `ClientWithState` protocol.
+        on
+            The event to trigger the bookmarking on.
```

Comment:
Do same for other docstring

```suggestion
            The event to trigger the bookmarking on. Supported values include:
            
                - `"response"` (the default): a bookmark is triggered when the assistant is done responding.
                - `None`: no bookmark is triggered
                
            When this method triggers a bookmark, it also updates the URL query string to reflect the bookmarked state.
```